src/gui/properties/peerlistsortmodel.h

```diff
@@ -48,10 +48,11 @@ class PeerListSortModel : public QSortFilterProxyModel
     bool lessThan(const QModelIndex &left, const QModelIndex &right) const
     {
         switch (sortColumn()) {
+        case PeerListDelegate::COUNTRY:
```

Comment:
wait... I think `COUNTRY` doesn't need `naturalCompare` the default comparison is sufficient.